accepted/2025/dotnet-package-download.md

```diff
@@ -0,0 +1,98 @@
+# Dotnet Package Download Command
+
+* Author: [Nigusu](https://github.com/Nigusu-Allehu)
+* GitHub Issues: [#12513](https://github.com/NuGet/Home/issues/12513), [#14491](https://github.com/NuGet/Home/issues/14491)
+
+## Summary
+
+This document proposes adding a `dotnet package download` command to the .NET CLI.
+The command would enable package download to a local folder scenarios without requiring a project file.
+
+## Motivation
+
+Developers often need to download NuGet packages outside of a project context:
+
+* Populating internal or offline feeds.
+* CI/CD workflows that consume `.nupkg` artifacts directly.
+* **Cross-platform builds** where many customers today rely on `nuget.exe install` in CI scripts running on Linux or macOS, forcing them to install Mono just to get this functionality.
+
+Providing `dotnet package download` improves developer productivity, simplifies pipelines, and ensures a cross-platform experience within the .NET CLI.
+
+## Explanation
+
+### Command Overview
+
+```ps1
+Description:
+  Downloads a NuGet package to a local folder without requiring a project file.
+
+Usage:
+  dotnet package download <PackageId> [options]
+
+Arguments:
+  PackageId    Package in the form of a package identifier (e.g. 'Newtonsoft.Json') 
+               or identifier and version separated by '@' (e.g. 'Newtonsoft.Json@13.0.3').
```

Comment:
NuGet.exe install doesn't have `packageID@version` format but instead has `-Version` option. [NuGet CLI install command | Microsoft Learn](https://learn.microsoft.com/nuget/reference/cli-reference/cli-ref-install#usage). Should we have an explicit `--version` option to this command.

Reply:
fwiw, we did talk about adding that conversation into the doc itself.